app/views/shares/index.html.erb

```diff
@@ -3,13 +3,25 @@
 <div class="col-xs-10 col-xs-offset-1 col-sm-8 col-sm-offset-2 col-md-6 col-md-offset-3">
   <% @shares.each do |share| %>
   <div class="booyah-box post">
-    <div class="handle"><%= current_user.name %></div>
-    <%= link_to share.title, share_path(share) %>
-    <p>
-      <%= linkify_hashtags(markdown(share.body)) %>
-    </p>
-    <%= pluralize(share.votes.count, "votes") %>
-    
+    <div class="handle">
+      <h4><%= link_to share.title, share_path(share) %></h4>
+      <p><%= linkify_hashtags(truncate(share.body, length: 100, separator: ' ')) %></p>
```

Comment:
This looks like it removes the markdown conversion that we had before.  I think we ultimately want both, although it's fairly complicated.

I think this gem may do it:
https://github.com/ianwhite/truncate_html